src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: isoscelesTrapezoid");
+        super.draw();
```

Comment:
The 'draw' method should provide all the necessary information about the figure, including its type, color, and unique attributes like width and height. Currently, it only prints the type and then calls 'super.draw()', which may not include all the needed details.